types/json-patch-gen/json-patch-gen-tests.ts

```diff
@@ -0,0 +1,24 @@
+import * as diff from "json-patch-gen";
```

Comment:
since this test is for the node usage, you should use the node-style import syntax: 

```ts
import diff = require("json-patch-gen")
```